app/assets/javascripts/alongslide/scrolling.coffee

```diff
@@ -368,7 +371,10 @@ class Alongslide::Scrolling
     skrollr.animateTo scrollTo * $(window).width(),
       duration: duration
       easing: 'easeOutQuad'
-      done: (skrollr) ->
+      done: (skrollr) =>
+        @stateData =
```

Comment:
Does this need to be a class variable? It doesn't appear to be read anywhere else.